config/base/env/config

```diff
@@ -10,7 +10,8 @@ REST_PORT=8080
 PROMETHEUS_PORT=9090
 TLS_HOSTNAME_OVERRIDE=
 TLS_PATH=/etc/tls
-NO_AUTH=false
+AUTH_DISABLE=false
```

Comment:
This absolutely needs to be called out as a breaking change (if we decide to accept it).

Reply:
I think this name makes it easier to categorise the options in the `config`, prepend with a short type description for the option.